crates/cli-tools/src/changelog.rs

```diff
@@ -58,6 +58,37 @@ impl Changelog {
         Self::parse(&String::from_utf8(fs::read(path).await?)?)
     }
 
+    async fn write_file(&self, path: &str) -> Result<()> {
+        fs::write(path, self.to_string().as_bytes()).await
+    }
+
+    fn push_description(&mut self, severity: &Severity, content: &str) -> Result<()> {
```

Comment:
`Severity` is `Copy`, you can take it by value
```suggestion
    fn push_description(&mut self, severity: Severity, content: &str) -> Result<()> {
```